src/test/resources/output/ServiceStoppedEvent.json

```diff
@@ -1,44 +1,42 @@
-{
-    "meta": {
-        "id": "0f84e591-a7e4-40b0-8c00-5d504ed1187f",
-        "type": "EiffelServiceStoppedEvent",
-        "version": "1.0.0",
-        "time": 1512111567421,
-        "tags": ["tag1", "tag2"],
-        "source": {
-            "domainId": "domainID",
-            "host": "host",
-            "name": "name",
-            "serializer": {
-                "groupId": "com.github.Ericsson",
-                "artifactId": "eiffel-remrem-semantics",
-                "version": "0.3.1"
-            },
-            "uri": "http://java.sun.com/j2se/1.3/"
-        },
-        "security": {
-            "sdm": {
-                "authorIdentity": "test",
-                "encryptedDigest": "sample"
-            }
-        }
-    },
-    "data": {
-        "serviceinstance": "Sample",
-        "uri": "http://java.sun.com/j2se/1.3/",
-        "customData": [{
-            "key": "firstLog",
-            "value": 1234.0
-        }, {
-            "key": "otherLog",
-            "value": 1234.0
-        }]
-    },
-    "links": [{
-        "type": "CAUSE",
-        "target": "aaaaaaaa-bbbb-5ccc-8ddd-eeeeeeeeeee1"
-    }, {
-        "type": "ACTIVITY_EXECUTION",
-        "target": "aaaaaaaa-bbbb-5ccc-8ddd-eeeeeeeeeee2"
-    }]
+{
+	"meta": {
+		"id": "798a6ef2-dc06-4279-b071-3dce7362df26",
+		"type": "EiffelServiceStoppedEvent",
+		"version": "1.0.0",

```

Comment:
this version will not break the generate validation

Reply:
it will not break the generate validation. Current version of all events in eiffel-operation-repo are 1.0.0 only